colour/models/rgb/ycbcr.py

```diff
@@ -0,0 +1,359 @@
+
+-   :func:`rgb_to_YCbCr`
+-   :func:`YCbCr_to_rgb`
+-   :func:`rgb_to_YcCbcCrc`
+-   :func:`YcCbcCrc_to_rgb`
+
+References
+----------
+.. [1]  Wikipedia. YCbCr. Retrieved February 29, 2016, from
+        https://en.wikipedia.org/wiki/YCbCr
+.. [2]  Recommendation ITU-R BT.709-6(06/2015). Retrieved February 29, 2016,
+        from https://www.itu.int/dms_pubrec/itu-r/rec/bt/\
+        R-REC-BT.709-6-201506-I!!PDF-E.pdf
+.. [3]  Recommendation ITU-R BT.2020(08/2012). Retrieved February 29, 2016,
+        from https://www.itu.int/dms_pubrec/itu-r/rec/bt/\
+        R-REC-BT.2020-0-201208-S!!PDF-E.pdf
+"""
+
+import numpy as np
+
+from colour.utilities import tsplit, tstack, CaseInsensitiveMapping
+from colour.models.rgb import REC_2020_COLOURSPACE
+
+__author__ = 'Colour Developers'
+__copyright__ = 'Copyright (C) 2013 - 2016 - Colour Developers'
+__license__ = 'New BSD License - http://opensource.org/licenses/BSD-3-Clause'
+__maintainer__ = 'Colour Developers'
+__email__ = 'colour-science@googlegroups.com'
+__status__ = 'Development'
+
+__all__ = ['WEIGHT',
+           'RANGE',
+           'rgb_to_YCbCr',
+           'YCbCr_to_rgb',
+           'rgb_to_YcCbcCrc',
+           'YcCbcCrc_to_rgb']
+
+WEIGHT = (CaseInsensitiveMapping(
+        {'Rec.601': [0.299, 0.114],
+         'Rec.709': [0.2126, 0.0722],
+         'Rec.2020': [0.2627, 0.0593]}))
+"""
+List of preset luma weightings:
+    'Rec.601' : [0.299, 0.114]
+    'Rec.709' : [0.2126, 0.0722]
+    'Rec.2020' : [0.2627, 0.0593]
+"""
+
+RANGE = (CaseInsensitiveMapping(
+        {'legal_12': [256./4095, 3760./4095],
+         'legal_12_YC': [256./4095, 3760./4095, 256./4095, 3840./4095],
+         'legal_12_int': [256, 3760],
+         'legal_12_YC_int': [256, 3760, 256, 3840],
+         'legal_10': [64./1023, 940./1023],
+         'legal_10_YC': [64./1023, 940./1023, 64./1023, 960./1023],
+         'legal_10_int': [64, 940],
+         'legal_10_YC_int': [64, 940, 64, 960],
+         'legal_8': [16./255, 235./255],
+         'legal_8_YC': [16./255, 235./255, 16./255, 240./255],
+         'legal_8_int': [16, 235],
+         'legal_8_YC_int': [16, 235, 16, 240],
+         'full': [0.0, 1.0],
+         'YPbPr': [0.0, 1.0, -0.5, 0.5]}))
+"""
+List of preset ranges:
+    'legal_12' : [256./4095, 3760./4095]
+    'legal_12_YC' : [256./4095, 3760./4095, 256./4095, 3840./4095]
+    'legal_12_int' : [256, 3760]
+    'legal_12_YC_int' : [256, 3760, 256, 3840]
+    'legal_10' : [64./1023, 940./1023]
+    'legal_10_YC' : [64./1023, 940./1023, 64./1023, 960./1023]
+    'legal_10_int' : [64, 940]
+    'legal_10_YC_int' : [64, 940, 64, 960]
+    'legal_8' : [16./255, 235./255]
+    'legal_8_YC' : [16./255, 235./255, 16./255, 240./255]
+    'legal_8_int' : [16, 235]
+    'legal_8_YC_int' : [16, 235, 16, 240]
+    'full' : [0.0, 1.0]
+    'YPbPr' : [0.0, 1.0, -0.5, 0.5]
+"""
+
+
+def rgb_to_YCbCr(rgb, K=WEIGHT['Rec.709'], inRange=RANGE['full'],
```

Comment:
1. `rgb_to_YCbCr` should be written `RGB_to_YCbCr` for consistency with the API: https://github.com/colour-science/colour/blob/develop/colour/models/rgb/rgb_colourspace.py#L398
2. I don't mind having each argument on a separated line too please!
3. `inRange` --> `in_range`, now something important and which makes the point of using tuples even more relevant: `inRange` is getting bound to a list (mutable). It is something to avoid in Python, except if you intend using the side effects of doing so (like in the case of recursion for example): http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments